static/sass/_pattern_strip.scss

```
@@ -811,6 +811,11 @@
       clip-path: polygon(0% 0%, 100% 0%, 100% calc(100% - 4rem), 0% 100%);
       padding-bottom: 6rem;
     }
+    &--top-right {
+      clip-path: polygon(0% 0%, 100% calc(0% + 4rem), 100% 100%, 0% 100%);
```

```suggestion
      clip-path: polygon(0% 0%, 100% 4rem, 100% 100%, 0% 100%);
```

isn't `calc(0% + 4rem)` just `4rem`?

---

@ClementChaumel very true, I was just keeping the calc consistent with the other classes